content/en/data_jobs/airflow.md

```diff
@@ -67,11 +67,26 @@ To get started, follow the instructions below.
 
 3. Trigger an update to your Airflow pods and wait for them to finish.
 
+4. Optionally, set up Log Collection for correlating Task logs to DAG run executions in DJM. Correlation requires the logs directory to follow the [default log filename format][6]. 
```

Comment:
```suggestion
4. Optionally, set up log collection for correlating task logs to DAG run executions in Data Jobs Monitoring. Correlation requires the logs directory to follow the [default log filename format][6]. 
```

Reply:
Suggesting this update because `DJM` isn't defined anywhere else on the page. 

Optionally, you could instead update [line 21](https://github.com/DataDog/documentation/blob/70f893143f279068e744f91bb2386252334cfbab/content/en/data_jobs/airflow.md?plain=1#L21) to say:
`[Data Jobs Monitoring][1] (DJM) provides visibility into the performance and reliability of workflows run by Apache Airflow DAGs.`